mono/utils/mono-threads-windows.c

```diff
@@ -590,6 +590,16 @@ mono_threads_platform_exit (gsize exit_code)
 	ExitThread (exit_code);
 }
 
+void
+mono_threads_platform_terminate_thread (MonoThreadInfo* info, gsize exit_code)
+{
+	HANDLE handle;
+
+	handle = info->native_handle;
+	g_assert(handle);
+	TerminateThread(handle, exit_code);
```

Comment:
Should we assert OS return code here for success?

Reply:
> Should we assert OS return code here for success?

Dont follow

Reply:
```
BOOL result = TerminateThread(handle, exit_code);
g_assert (TerminateThread);
```

Reply:
Fixed